sdk/core/Azure.Core/tests/TestFramework/UseSyncMethodsInterceptor.cs

```diff
@@ -75,6 +75,10 @@ public void Intercept(IInvocation invocation)
 
             try
             {
+                if (methodInfo.ContainsGenericParameters)
```

Comment:
Can we make sure to add tests for new features to https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core/tests/ClientTestBaseTests.cs

Reply:
Added and commented better